docker-compose.override.yml

```diff
@@ -4,7 +4,7 @@ services:
 ###> doctrine/doctrine-bundle ###
   database:
     ports:
-      - "5432"
+      - "5432:5432"
```

Comment:
this shouldn't be needed. Without this, it exposes to a random port on your host. Then you use `symfony console some:command`, which allows the symfony binary to expose the `DATABASE_URL` env var containing whatever that port is.

Reply:
correct - so do we want to "force" the use of the symfony binary when running commands? e.g. anything w/ makerbundle thats doctrine related? 

We could add a note on screen, hey if you want to use `bin/console .... ` or look at the database using PHPStorms database tool, you may need to expose this to a specific port. (what I usually do)

Reply:
> do we want to "force" the use of the symfony binary when running commands? e.g. anything w/ makerbundle thats doctrine related?

Yes. We basically say "Use our Docker setup, and then always run `symfony console` or configure your own DB manually without Docker"